app/src/main/java/org/p2p/wallet/alarmlogger/model/StrigaAlarmError.kt

```diff
@@ -0,0 +1,7 @@
+package org.p2p.wallet.alarmlogger.model
+
+data class StrigaAlarmError(
+    val source: String,
+    val kycSdkState: String,
+    val error: String
```

Comment:
I think you can put Throwable in here and then make it String in converter

the same logic you have in SendConverter